### DISCUSSION TITLE
征集【误差图 · Error Bar】贡献者 🎉

### DISCUSSION THREAD

**Discussion by charxie** [antvis/G2Plot#2326] [Ideas]
### Line plots seem to miss error bar.

Is there a way to draw error bars in line plots? (This is not exactly a bug report.)

**Comment by visiky** [comment]
maybe you're interested in this plot: https://g2.antv.vision/en/examples/other/other#column-error ?

you can try to create a custom plot:

1, using custom plugins:  https://g2plot.antv.vision/en/docs/manual/plugin
2, using multi-view: https://g2plot.antv.vision/en/docs/api/multi-view

otherwise, you can use `annotations` to draw the error bars

**Comment by charxie** [comment]
Thanks! Using annotations seem to be a good choice.

**Reply by charxie** [reply]
I am a TypeScript developer. I may send you a pull request if I figure out an optimal way of adding the error bar support to G2Plot. This is an essential feature for a graph to the science and engineering community.
